README.md

```diff
@@ -101,45 +99,140 @@ The back-end service implements the following protocol:
 
 The `POST` endpoints accept the following JSON body:
 
-```
+```ts
 { 
   "filters": [{
-    "value": string or { id: string },
-    "fields": string[] (field id's)
+    "fields": string[], /* field id's */
+    "operator": string, /* '=' or 'ILIKE' */
+    "value": string|{ id: string }
   }, ...],
   "sort": {
-    "path": string (field id),
+    "path": string /* field id */,
     "ascending": boolean
   }
 }
 ```
 
 And they return the following response:
 
-```
+```ts
```

Comment:
Shouldn't this be JSON?

Reply:
fails to format well, because JSON cannot have comments. I have switched it to js instead of ts, that looks ok.